plugins/com.google.cloud.tools.eclipse.appengine.localserver/META-INF/MANIFEST.MF

```diff
@@ -33,7 +33,8 @@ Require-Bundle: com.google.cloud.tools.app.lib;bundle-version="0.1.0",
  org.eclipse.m2e.jdt;bundle-version="1.6.2",
  org.eclipse.ui.console;bundle-version="3.6.100",
  org.eclipse.ui.ide;bundle-version="3.11.0",
- org.eclipse.jdt.core
+ org.eclipse.jdt.core,
+ com.google.cloud.tools.eclipse.preferences
```

Comment:
1. Could we make this an Import-Package instead
2. Nit: sort with the other com.google packages